engine/access/rpc/backend/backend.go

```diff
@@ -145,6 +148,16 @@ func New(params Params) (*Backend, error) {
 		}
 	}
 
+	sporkRootBlockHeight, err := params.State.Params().SporkRootBlockHeight()
```

Comment:
I wonder where do we get this data from? Did we already implemented the snapshot generation which includes the spork root block?

Reply:
`Params.SporkRootBlockHeight` is included in the [mainnet23 root-protocol-state-snapshot.json](https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-23-execution/public-root-information/root-protocol-state-snapshot.json). It's extracted from the root snapshot loaded on boot, then written into the db here:
https://github.com/onflow/flow-go/blob/master/state/protocol/badger/state.go#L556-L563